common/yak/php/php2ssa/visit_expr.go

```diff
@@ -427,13 +438,27 @@ func (y *builder) VisitExpression(raw phpparser.IExpressionContext) ssa.Value {
 
 	case *phpparser.ShortQualifiedNameExpressionContext:
 		//因为涉及到函数，先peek 如果没有读取到说明是一个常量 （define定义的常量会出现问题）
-		identifier := y.VisitIdentifier(ret.Identifier())
-		if value := y.ir.PeekValue(identifier); value != nil {
+		var unquote string
+		_unquote, err := yakunquote.Unquote(ret.Identifier().GetText())
+		if err != nil {
+			unquote = ret.Identifier().GetText()
+		} else {
+			unquote = _unquote
+		}
+		//先在常量表中查询
+		if !y.isFunction {
+			s, ok := y.constMap[unquote]
+			if ok {
+				return y.ir.EmitConstInst(s)
```

Comment:
should define `constMap map[string]ssa.Value`,and just return `s` here.
this code will create multiple const value. 
```php
const a = 1;  // t0 = 1
println(a)
// should: println(t0)
// actual: t1=1; println(t1);
println(a)
// should: pritnln(t0)
// actual: t2=1; println(t2);
```